examples/ibm-lbaas/main.tf

```diff
@@ -0,0 +1,123 @@
+fgvJf6GkcJ/JnAi8qhH5IV50/X3YWdosY2fGBzR7Naasfh4IrNq6tZ+1L5c/6agP
+RfKU+0ECgYEA+t6fPgcE211inH4H0i8H5HrI/sgmsF7uXiobbcUCFBJR3rT8XUq0
+9x7SEj5CokvpDm1pM3ktv/fffB2W74pcpn63n8rWjHOu3/LMvnab8Ad54wA7IMF8
+/vvjhbqZaWhbYt93o5bFP6U3QlfLaMRItr+0KLm7kyJ4GBC6QGRSDhECgYEA5Ovh
+oBILLZriVcuVwYeLxuzjCCJohpFkUtXmxUpwLKYRVAsC0MSNTjvZfJkVOvR9G8Ki
+Cmy7wGt1VIo8M7DKmetHTsXn6H9S0SN62ykKX/ob/D1g0tFETsEFkVt7mha3Q1AB
+6VR9LiohCQAevoOLn+Vm8B4aHyOGjah2FgPta0sCgYAN3lbBUBQFqID2E8WM6gqu
+p9cKtrfk0iqtS/ieNeDqiSS7ghfddG7SpoKIfaajYDzvDj9dmBpeXW6eZuhcL7L1
+hVXTYJxBwXdua/bDpLz0JQWo9e9O3UNyuSwXzXwDpsA+lAoCIiifXxvR8BaPoSI/
+8BMemT30YVhwRCR3wNQEcQKBgQCwcULRTrcA6p1DBYyiwuewZotCjMrF1bBezHF3
+ZT16nHFEtsvvv18uiqDCEXe0nhcD24trv40i7XBcvcNTEBPIePjYNV/e6qwZeGBM
+JaDSgwMo8uH6+8LLdKjm9X0aMiIEptkiT7XAbEZUGpyXuOpYTsd9kaYOlCI0c0C5
+DUPkawKBgGlwzHX3dr7jYldmB9/g94jWeNkX6KPtSDNaKZ9WzIuywCB6wua7AVXa
+NXMjAHErbX2J+8k85TccHR1ps3MgBbFHdiuJwx2vUPLfVj53GWUXmg4Gw4zUs5mq
+ykXbeuyhK6AL6V3NsJyP454bM8dmZnxBrZvRo5FnqQInGgwGSjgc
+-----END RSA PRIVATE KEY-----
+    EOF
+}
+
+resource "ibm_compute_ssh_key" "ssh_public_key_for_app-vms" {
+  label      = "${var.ssh_label}"
+  notes      = "${var.notes}"
+  public_key = "${file(var.ssh_key_path)}"
+}
+
+# Create a new virtual guest using image "UBUNTU_16_64"
+resource "ibm_compute_vm_instance" "vm_instances" {
+  count                    = "2"
+  hostname                 = "${format("lbaas-example-%02d", count.index + 1)}"
+  os_reference_code        = "${var.osref}"
+  domain                   = "${var.domain}"
+  datacenter               = "${var.datacenter}"
+  network_speed            = "10"
+  hourly_billing           = true
+  private_network_only     = false
+  cores                    = "1"
+  memory                   = "1024"
+  disks                    = ["25"]
+  user_metadata            = "{\"value\":\"newvalue\"}"
+  dedicated_acct_host_only = true
+  post_install_script_uri  = "${var.vm-post-install-script-uri}"
+  local_disk               = false
+  ssh_key_ids              = ["${ibm_compute_ssh_key.ssh_public_key_for_app-vms.id}"]
+}
+
+resource "ibm_lbaas" "lbaas" {
+  name        = "${var.name}"
+  description = "lbaas example"
+  subnets     = ["${var.subnet_id}"]
+  datacenter  = "${var.datacenter}"
+
+  protocols = [{
+    frontend_protocol     = "HTTPS"
+    frontend_port         = 443
+    backend_protocol      = "HTTP"
+    backend_port          = 80
+    load_balancing_method = "${var.lb_method}"
+    tls_certificate_id    = "${ibm_compute_ssl_certificate.lbaas-ssl-certificate.id}"
+  },
+    {
+      frontend_protocol     = "HTTP"
+      frontend_port         = 80
+      backend_protocol      = "HTTP"
+      backend_port          = 80
+      load_balancing_method = "${var.lb_method}"
+    },
+  ]
+
+  server_instances = [{
+    private_ip_address = "${ibm_compute_vm_instance.vm_instances.0.ipv4_address_private}"
+  },
+    {
+      private_ip_address = "${ibm_compute_vm_instance.vm_instances.1.ipv4_address_private}"
+    },
+  ]
+}
```

Comment:
Can we add a output variable and update readme to look into the value of this output variable to access the application behind the lb.

Reply:
Done. Added outputs.tf